server/Models/Recipe.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace server.Models
+{
+    public class Recipe
+    {
+        public required int Id { get; set; }
+        public required string Name { get; set; }
+        public required string Description { get; set; }
+        public required string Ingredients { get; set; }
+        public required string Directions { get; set; }
+        public string? Image { get; set; }
+        public required DateTime Date_Created { get; set; }
+        public required DateTime Date_Updated { get; set; }
+        public string? Categories { get; set; }
+        public string? Cook_Time { get; set; }
```

Comment:
TODO: figure out how to map these values properly. If nothing works then we'll rename the database columns.